### ISSUE TITLE
FileUploader: add `required` and `accessibleNameRef`

### ISSUE THREAD

**Issue by Lukas742** [SAP/ui5-webcomponents#5893]
### **Feature Request Description**
For screen reader support we need to be able to mark every input-like component as required and also add a label to them. For this we would need the `FileUploader` to accept `required` and `accessibleNameRef`. 

### **Proposed Solution**
Add `required` and `accessibleNameRef` to the component's public properties/attributes.

### **Additional Context**
We need this for our [FilterBar](https://sap.github.io/ui5-webcomponents-react/?path=/docs/layouts-floorplans-filterbar--default-story) implementation. There it's possible to set a label for a filter and also make it mandatory (required). Per default screen readers don't read the label on top of the filter (for that we need `accessibleNameRef`) and if the filter is required, the input should be required as well (for that we need `required`).

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

A clear and concise description of the impact/urgency of the required feature.

### **Stakeholder Info** (if applicable)

- Organization: UI5 Web Components for React

**Comment by DMihaylova** [comment]
Hello @Lukas742,

We'll start working on adding _accessibleNameRef_ while for _required_ there are internal discussions that are currently taking place.

Regards,
Diana

**Comment by DMihaylova** [comment]
Hello @Lukas742,

We implemented _accessibleNameRef_.
The reason for the delay of this issue is because together with the design colleagues we are working on a potential redesign of the component's look. That's a prerequisite for implementing _required_.
